api/api/clubs/views.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from .models import Club
+from .serializers import ClubsSerializer
+
+
+@api_view()
+def clubs_details(request):
```

Comment:
Let's prefer class based views for consistency.
https://github.com/BitByte-TPC/gymkhana/blob/accc96ee7be245e82536f3bc7657e9ceb183c7c6/api/api/auth/views.py#L18
https://www.django-rest-framework.org/api-guide/views/#class-based-views